src/com/akshay/captain/jira.main.kts

```diff
@@ -39,21 +39,28 @@ private val bugFixes = "Bug fixes"
 private val instrumentation = "Instrumentation"
 private val labels = "Labels"
 
+//env vars
+private val userToken = "userToken"
+private val userName = "userName"
+
 //Jira Variables
 private val clueJiraUrl = "https://biowink.atlassian.net"
 private val readyForReleaseFilter = "filter=15768"
 
 main()
 
 fun main() {
-    val platform = args.firstOrNull()?.toPlatform() ?: Platform.Android
 
-     createReleaseVersion()
-    val cred = BasicCredentials("","")
+    val platform = args.firstOrNull()?.toPlatform() ?: Platform.Android
+    val cred = BasicCredentials(System.getenv()[userName], System.getenv()[userToken])
     val jira = JiraClient(clueJiraUrl, cred)
     val releaseNotes = mutableListOf<String>()
+    val webhookPath = args.lastOrNull() ?: error("Missing path to the source CSV file")
```

Comment:
wishlist (maybe future iteration): webhook URLs shouldn't be passed every time the user runs this script, can't we have them as optional and have a default empty file inside the project that people can just replace?

Super amazing: make this repo private and add have that default webhook url file with the values already there instead of empty. 

The main objective of all these wishlist points is to reduce the pain of manual work from devs, specially for these manual steps (like pasting the same webhooks file and getting the csv file from the same filter in Jira.)